index.html

```diff
@@ -0,0 +1,304 @@
+                  ><img src="icons/medium.png" alt="medium icon"
+                /></a>
+              </li>
+              <li class="headline-section__social--item">
+                <a href="#" class="social-icon github"
+                  ><img src="icons/github.svg" alt="github icon"
+                /></a>
+              </li>
+              <li class="headline-section__social--item">
+                <a href="#" class="social-icon chill"
+                  ><img src="icons/angelist.png" alt="angelist icon"
+                /></a>
+              </li>
+            </ul>
+
+            <button type="button" class="btn resume">Get my resume</button>
+          </div>
+          <ul class="language-section-container">
+            <li class="language-section open">
+              <div class="language-title-section">
+                <button type="button" class="drop-down down">Language</button>
+                <img src="./icons/expanded.svg" alt="expanded arrow" />
+              </div>
+
+              <ul class="language-section-box-sec">
+                <li class="language-section-box">
+                  <img src="./images/pic1.png" alt="javaScript logo" />
+                  <p class="language-name">JavaScript</p>
+                </li>
+                <li class="language-section-box">
+                  <img src="./images/pic2.png" alt="html logo" />
+                  <p class="language-name">HTML</p>
+                </li>
+                <li class="language-section-box">
+                  <img src="./images/pic3.png" alt="css logo" />
+                  <p class="language-name">Css</p>
+                </li>
+              </ul>
+            </li>
+            <li class="language-section">
+              <div class="language-title-section">
+                <button type="button" class="drop-down down">Framework</button>
+                <img src="./icons/not-expanded.svg" alt="not-expanded arrow" />
+              </div>
+            </li>
+            <li class="language-section">
+              <div class="language-title-section">
+                <button type="button" class="drop-down down">Skills</button>
+                <img src="./icons/not-expanded.svg" alt="not-expanded arrow" />
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <!-- contact-section -->
+      <section class="contact-form">
+        <div class="contact-form-section">
+          <div class="contact-form-card">
+            <h2 class="contact-form-title">Contact me</h2>
+            <p class="contact-form-description">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+          <form id="form" action="ttps://formspree.io/f/mjvlgvab" method="post">
+            <label for="username"
```

Comment:
[REQUIRED] Please kindly note that when using the `<label/>` tag, we need to specify the `id` of the `<input/>`, which we are reffering to, in the `for` attribute. Please refer the  sample code below:
```HTML
 <label for="userName">Full name</label>
  <input type="text" id="userName">
```
In your case, you are nesting your `<inputs/>` correctly but  specifying `"username"` in the `for` attribute of your `<label/>` and the **id** of the respective `<input/>` is set to `"user-name"`. That is why you are not passing [webhint](https://github.com/mesfin345/Portfolio/runs/5214165232?check_suite_focus=true) check off your `linters`.

- [ ] Please kindly resolve the issue so we can pass all linter checks